Source/CarthageKit/Project.swift

```diff
@@ -542,31 +464,29 @@ public final class Project {
 						return self.createVersionFilesForFrameworks(frameworkURLs, fromDirectoryURL: directoryURL, projectName: projectName, commitish: commitish)
 					}
 					.then(SignalProducer<URL, CarthageError>(value: directoryURL))
-		}
+			}
 	}
 
 	/// Removes the file located at the given URL
 	///
 	/// Sends empty value on successful removal
 	private func removeItem(at url: URL) -> SignalProducer<(), CarthageError> {
 		return SignalProducer<URL, CarthageError>(value: url)
-			.attemptMap({ (url: URL) -> Result<(), CarthageError> in
+			.attemptMap { url in
 				do {
 					try FileManager.default.removeItem(at: url)
 					return .success()
```

Comment:
Sorry, know that this compiles, but there's sometimes hard-to-diagnose behavior surrounding this: need `return .success(())` here.